completion/zsh/_task

```diff
@@ -55,13 +63,13 @@ _task() {
         '(-t --taskfile)'{-t,--taskfile}'[specify a different taskfile]:taskfile:_files' \
         '(-v --verbose)'{-v,--verbose}'[verbose mode]' \
         '(-w --watch)'{-w,--watch}'[watch-mode for given tasks, re-run when inputs change]' \
+        '(operation)*: :__task_list' \
         + '(operation)' \
```

Comment:
Task-name completions didn't work except for the first positional argument in zsh, but this has been fixed as well.

(Argument specifications after `+ '(operation)'` become mutually exclusive options. In the previous code, `*: :__task_list` also prevented other task names from being suggested when one task name is specified in a command line.)